lib/node_modules/@stdlib/utils/homedir/test/test.homedir.js

```diff
@@ -0,0 +1,364 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var proxyquire = require( 'proxyquire' );
+
+
+// VARIABLES //
+
+var mpath = './../lib/homedir.js';
+
+
+// FUNCTIONS //
+
+/**
+* Returns a copy of the default proxyquire options.
+*
+* @private
+* @returns {Object} options object
+*/
+function makeOpts() {
+	var opts = {
+		'@stdlib/utils/platform': 'darwin',
+		'@stdlib/utils/is-windows': false
+	};
+	return opts;
+} // end FUNCTION makeOpts()
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	var homedir;
+	t.ok( true, __filename );
+	homedir = require( mpath );
+	t.strictEqual( typeof homedir, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function supports older Node versions', function test( t ) {
```

Comment:
How does this test whether the implementation supports older Node versions? Considering how this package works, this should be in the other test file.